package.json

```diff
@@ -82,7 +82,7 @@
         "formik": "^2.1.4",
         "husky": "^3.1.0",
         "jest": "^24.9.0",
-        "jest-canvas-mock": "^2.2.0",
+        "jest-canvas-mock": "2.4.0",
```

Comment:
Do we need this specific version or can we use `^` here?

Reply:
@jstoffan we can use `^`